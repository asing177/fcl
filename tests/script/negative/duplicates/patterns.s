type T = C(int f1, bool f2);

bool b;

@initial
f(T t) {
  b = case t { C(x,x) -> x };
  terminate()
}