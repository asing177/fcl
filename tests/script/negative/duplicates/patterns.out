Error at Line 7, Column 18:
   The binding 'x' shadows the variable bound by a pattern
      of the same name, of type 'bool',
      bound at Line 7, Column 20.